### PR TITLE
Feature: Edit Settings Page

### PR DESCRIPTION
**Pull request by RakeshUP**
## Description

The Edit DAO Settings page is set up, and right now it's empty with no form functionality. 
Fetching and hydrating the data for the form and form validation will be done as a separate task.

Task: [425](https://aragonassociation.atlassian.net/jira/software/c/projects/APP/boards/41?modal=detail&selectedIssue=APP-425)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [ ] I have tested my code on the test network.

### CODE REVIEW

File: packages/ui-components/src/components/wizard/wizard.tsx [hunk 1]
```diff
@@ -3,12 +3,12 @@ import styled from 'styled-components';
 import {LinearProgress} from '../progress';
 
 export type WizardProps = {
```
Comment (Fabricevladimir) [inline packages/ui-components/src/components/wizard/wizard.tsx:5]: So apparently this is a whole new ui-component. Header -> Page. While there is a page-header component in web-app, it needs to be refactored. I'll leave it up to you to decide whether to extend it or create a new component.

File: packages/web-app/src/pages/editSettings.tsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+            title={t('settings.editDaoSettings')}
+            description={t('settings.editSubtitle')}
+            nav={
+              <Breadcrumb icon={icon} crumbs={breadcrumbs} onClick={navigate} />
+            }
+          />
+        </div>
+
+        <div>
+          <Accordion>
```
Comment (Fabricevladimir) [inline packages/web-app/src/pages/editSettings.tsx:68]: The space between the accordion sections should be `space-y-3` and not the same as the parent container `space-y-8`

File: packages/web-app/src/pages/editSettings.tsx [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+  const {breadcrumbs, icon} = useMappedBreadcrumbs();
+  const formMethods = useForm({
+    mode: 'onChange',
+    defaultValues,
+  });
+
+  return (
+    <FormProvider {...formMethods}>
+      <Container>
+        <div className="-mx-2 desktop:mx-0">
```
Comment (Fabricevladimir) [inline packages/web-app/src/pages/editSettings.tsx:44]: The header has the edit settings button on mobile.

File: packages/web-app/src/pages/editSettings.tsx [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+          {currentMenu === 'governance' && (
+            <AccordionContent>
+              <ConfigureCommunity />
+            </AccordionContent>
+          )}
+        </div>
+
+        <ButtonContainer>
+          <HStack>
+            <ButtonText
```
Comment (Fabricevladimir) [inline packages/web-app/src/pages/editSettings.tsx:121]: On mobile the buttons should be of size `large`

**Review by Fabricevladimir — COMMENTED** [review]
Nice work, thanks! Left a few comments

File: packages/web-app/src/pages/editSettings.tsx (hunk 3 above)
Comment (sepehr2github) [reply packages/web-app/src/pages/editSettings.tsx:44]: Also, disabled reset all changes!

**Review by sepehr2github — APPROVED** [review]
Thanks, Rakash! Just needs a UI review. Fabrice mentioned most of them

File: packages/web-app/src/pages/editSettings.tsx (hunk 2 above)
Comment (RakeshUP) [reply packages/web-app/src/pages/editSettings.tsx:68]: The space between the accordion sections is 64px on desktop and 40px on mobile. I guess only the space between form items is space-y-3

File: packages/ui-components/src/components/wizard/wizard.tsx (hunk 1 above)
Comment (RakeshUP) [reply packages/ui-components/src/components/wizard/wizard.tsx:5]: I'll refactor this in my next PR. That too is related to the settings page

**Comment by Fabricevladimir** [comment]
Noticed that on the Settings page on mobile, the Propose New Settings button in the header doesn't navigate 🙈

**Review by Fabricevladimir — APPROVED** [review]

**Pull request merged**